extensions/registry.py

```diff
@@ -351,4 +351,9 @@
         'flags' : { 'public' },
         'url' : 'extensions/amd/cl_amd_planar_yuv.txt',
     },
+    'cl_pocl_content_size' : {
+        'number' : 71,
+        'flags' : { 'public' },
+        'url' : 'extensions/pocl/cl_pocl_content_size.txt',
```

Comment:
```suggestion
        'url' : 'extensions/pocl/cl_pocl_content_size.html',
```

I think this should have a html file extension instead?

You'll need to rebuild clext.php after making this change.